include/config.h

```diff
@@ -38,11 +38,17 @@ THE SOFTWARE.
 #define BOARD_cantact     2
 #define BOARD_canable     3
 #define BOARD_usb2can     4
-#define BOARD_canalyze     5
+#define BOARD_canalyze    5
 #define BOARD_cannette    6
 
+#ifdef VERSION_STR
+    #define PRODUCT_APPEND_STR " " VERSION_STR
+#else
+    #define PRODUCT_APPEND_STR ""
```

Comment:
The `""` doesn't do anything, and it's kind of confusing to have it.

Reply:
Removed ""